documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -221,16 +208,29 @@ data:
   user.password: _PASSWORD-PROTECTING-P12-ARCHIVE_
 ----
 
-. Configure your client to connect to the Kafka cluster with the properties required to make a secure connection to the Kafka cluster.
+. Extract the public cluster CA certificate in PKCS #12 format:
```

Comment:
We wouldn't extract them in PKCS #12 format if they were in PEM format? Should we say the command would be similar for PEM? Extract according to format. This example shows PKCS format...